src/changelog/.10.x.x/change_default_module_name.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<entry xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xmlns="http://logging.apache.org/log4j/changelog"
+       xsi:schemaLocation="http://logging.apache.org/log4j/changelog https://logging.apache.org/log4j/changelog-0.1.1.xsd"
+       type="changed">
+  <author id="github:ppkarwasz"/>
+  <description format="asciidoc">Changes the default OSGi and module name to use full stops `.` instead of non-alphanumeric characters.</description>
```

Comment:
```suggestion
  <description format="asciidoc">Changes the default OSGi and module name to use full stops `.` instead of non-alphanumeric characters</description>
```